### PR TITLE
feat(copy): fix detecting tmux & upgrade go-osc52 to v2

### PR DESCRIPTION
**Pull request by aymanbagabas**

### CODE REVIEW

File: copy.go [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+)
 
 // Copy copies text to clipboard using OSC 52 escape sequence.
 func (o Output) Copy(str string) {
-	o.osc52Output().Copy(str)
+	s := osc52.New(str)
+	if strings.HasPrefix(o.environ.Getenv("TERM"), "screen") {
+		s = s.Screen()
+	}
+	s.WriteTo(o)
```
Comment (muesli) [inline copy.go:15]: Should discard the error returned from `WriteTo`.

File: copy.go [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
 
 // CopyPrimary copies text to primary clipboard (X11) using OSC 52 escape
 // sequence.
 func (o Output) CopyPrimary(str string) {
-	o.osc52Output().CopyPrimary(str)
+	s := osc52.New(str).Primary()
+	if strings.HasPrefix(o.environ.Getenv("TERM"), "screen") {
+		s = s.Screen()
+	}
+	s.WriteTo(o)
```
Comment (muesli) [inline copy.go:25]: Should discard the error returned from `WriteTo`.

File: copy.go [hunk 3]
```diff
@@ -1,22 +1,28 @@
...
-	return osc52.NewOutput(o.tty, o.environ.Environ())
-}
+	"github.com/aymanbagabas/go-osc52/v2"
+)
 
 // Copy copies text to clipboard using OSC 52 escape sequence.
 func (o Output) Copy(str string) {
-	o.osc52Output().Copy(str)
+	s := osc52.New(str)
+	if strings.HasPrefix(o.environ.Getenv("TERM"), "screen") {
```
Comment (muesli) [inline copy.go:12]: Isn't that a logic that should be part of the osc52 package itself? But I guess it doesn't access the environment any longer?

Comment (aymanbagabas) [reply copy.go:12]: Yea, it doesn't access the environment any more.

**Pull request merged**